apps/web/components/Templates/PageNavigation.tsx

```diff
@@ -0,0 +1,40 @@
+import Link from "next/link";
+
+export default function PageNavigation() {
+  function switchPage() {
+    // Coming soon...
+  }
+
+  return (
+    <section className="w-full">
+      <div className="flex h-5 justify-center md:h-8">
+        <Link href="">
+          <a className="flex w-5 items-center justify-center rounded bg-sky-500 p-3 text-xs font-medium transition-colors hover:bg-sky-400 md:w-8 md:text-base">
+            1
+          </a>
+        </Link>
+      </div>
+
+      <div className="mt-8">
+        <p className="relative text-xs text-zinc-300 before:absolute before:right-[55%] before:left-0 before:top-1/2 before:h-px before:bg-zinc-200/10 before:content-[''] after:absolute after:left-[55%] after:right-0 after:top-1/2 after:h-px after:bg-zinc-200/10 after:content-[''] md:text-base">
```

Comment:
Getting rid of before/after and puting those into markup would probably make things much more readable.

Before/After are just weird in TW imo, it would prolly be better to simply add those elements. It doesn't have any downside to it.